src/main/java/baseball/Util.java

```diff
@@ -0,0 +1,59 @@
+package baseball;
+
+import camp.nextstep.edu.missionutils.Console;
+import camp.nextstep.edu.missionutils.Randoms;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class Util {
+
+    public static final int BASE_COUNT = 3;
```

Comment:
```suggestion
    private static final int BASE_COUNT = 3;
```

매직넘버의 상수화 매우 좋습니다 💯 
하지만 접근제어자가 public 으로 되어있네요.  모든 클래스에 있는 필드변수들을 최대한 private으로 닫아주는게 좋아요 캡슐화와 응집도, 결합도라는 것과 관련이 있는데, 지금은 이해하기 어려울 수 있으니 슬쩍 한번 읽어보시는것도 좋아 보입니다 :)

https://bperhaps.tistory.com/entry/%EC%BA%A1%EC%8A%90%ED%99%94%EB%9E%80-%EB%AC%B4%EC%97%87%EC%9D%B8%EA%B0%80-%EC%96%B4%EB%96%A4-%EC%9D%B4%EC%A0%90%EC%9D%B4-%EC%9E%88%EB%8A%94%EA%B0%80

BASE_COUNT를 다른곳에서도 사용하는 것 같은데, private으로 닫으면 다른곳에서는 이 상수를 사용 못하겠네요. 그렇다면 다른 방법은 무엇이 있을까요? 한번 생각해보죠 :)

Reply:
다른 곳에서 사용하기 위해 public으로 선언을 했던 것인데,

오히려 private으로 닫고 다른 방식을 이용할 수 있는 부분을 생각치 못했네요!!

상수를 캡슐화한 후에 재이용하는 방식을 찾아보겠습니다ㅎㅎ